stash_engine/lib/tasks/counter/log_combiner.rb

```diff
@@ -0,0 +1,76 @@
+require 'net/scp'
+require 'time'
+
+module Counter
+  class LogCombiner
+
+    USERNAME = 'dryad'.freeze
+    PRIMARY_FN_PATTERN = /^counter_\d{4}-\d{2}-\d{2}.log$/
+    ANY_LOG_FN_PATTERN = /^counter_(\d{4})-(\d{2})-(\d{2}).log(|_\d{1}|_combined)$/
+
+    def initialize(log_directory:, scp_hosts:, scp_path:)
+      @log_directory = log_directory
+      @scp_hosts = scp_hosts
+      @scp_path = scp_path
+      @scp_path = scp_path
```

Comment:
looks like you only need one of these lines

Reply:
:-)  I must've accidentally duplicated the line.  There is something in my editor that I hid on accident that does that sometimes.